mhbuehler/GenAIExamples
# UI Audio Upload

## Description

Enables audio file upload and real-time recording upload capability through the gradio UI. Also reduces some duplicated code, improves display text, and suppresses the PDF ingestion tab.

## Issues

[MultimodalQnA Enhancements RFC](https://github.com/opea-project/docs/pull/208)

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [x] Others (enhancement, documentation, validation, etc.)

## Dependencies

No new dependencies

## Tests

Manually tested the interface:

- Two types of audio ingestion/upload (file upload & real-time recording)
- Subsequent query to verify that audio was accurately transcribed and saved
